pkg/lib/serverTime.js

```diff
@@ -258,42 +266,43 @@ export function ServerTime() {
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_os_release() {
+        if (self.os_release)
+            return Promise.resolve(self.os_release);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20221213-115634-adc42487-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)